### PR TITLE
documentation: Fix readthedocs builds #572

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

File: lib/rucio/core/trace.py [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Mario Lassnig <mario.lassnig@cern.ch>, 2013
+# - Thomas Beermann <thomas.beermann@cern.ch>, 2014-2017
+# - Vincent Garonne <vgaronne@gmail.com>, 2015-2018
 # Copyright European Organization for Nuclear Research (CERN)
```
Comment (mlassnig) [inline lib/rucio/core/trace.py]: duplicate addition?

File: lib/rucio/core/nongrid_trace.py [hunk 2]
```diff
@@ -28,22 +34,31 @@
...
-HANDLER.suffix = "%Y-%m-%d"
-LOGGER.addHandler(HANDLER)
+try:
+    HANDLER = logging.handlers.RotatingFileHandler(filename='%s/trace' % config_get('nongrid-trace', 'tracedir'), maxBytes=1000000000, backupCount=10)
+    LOGFORMATTER = logging.Formatter('%(message)s')
+    HANDLER.setFormatter(LOGFORMATTER)
+    HANDLER.suffix = "%Y-%m-%d"
+    LOGGER.addHandler(HANDLER)
+except:
+    if not any("sphinx-build" in argc for argc in sys.argv):
```
Comment (mlassnig) [inline lib/rucio/core/nongrid_trace.py]: better check seems to be  ``if 'sphinx' not in sys.modules:``

File: bin/rucio-cache-client [hunk 3]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```
Comment (bari12) [inline bin/rucio-cache-client]: Why remove it in just this file?

Comment (vingar) [reply bin/rucio-cache-client]: mistake. phew another hour of tests running in travis :(

**Comment by bari12** [comment]
`lib/rucio/client/baseclient.py:68:5: F811 redefinition of unused 'HTTPKerberosAuth' from line 53`
This might be due to my manual merge.

**Comment by vingar** [comment]
I've fixed it.

**Review by bari12 — APPROVED** [review]

**Pull request merged**